content/en/user-guide/integrations/crossplane/index.md

```diff
@@ -0,0 +1,226 @@
+
+In the following, we first install the AWS provider for S3.
+Please note that you can copy/paste the entire multi-line command below into your terminal:
+{{<command>}}
+$ cat <<EOF | kubectl apply -f -
+apiVersion: pkg.crossplane.io/v1
+kind: Provider
+metadata:
+  name: provider-aws-s3
+spec:
+  package: xpkg.upbound.io/upbound/provider-aws-s3:v0.40.0
+EOF
+{{</command>}}
+
+We also install the AWS provider for SQS:
+{{<command>}}
+$ cat <<EOF | kubectl apply -f -
+apiVersion: pkg.crossplane.io/v1
+kind: Provider
+metadata:
+  name: provider-aws-sqs
+spec:
+  package: xpkg.upbound.io/upbound/provider-aws-sqs:v0.40.0
+EOF
+{{</command>}}
+
+After some time, the providers should get into healthy state, which can be confirmed via `kubectl get providers`:
+{{<command>}}
+$ kubectl get providers
+NAME                          INSTALLED   HEALTHY   PACKAGE                                               AGE
+upbound-provider-family-aws   True        True      xpkg.upbound.io/upbound/provider-family-aws:v0.40.0   2m
+provider-aws-s3               True        True      xpkg.upbound.io/upbound/provider-aws-s3:v0.40.0       2m
+provider-aws-sqs              True        True      xpkg.upbound.io/upbound/provider-aws-sqs:v0.40.0      2m
+{{</command>}}
+
+Next, we install a secret to define the test credentials for the AWS provider:
+{{<command>}}
+$ cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: localstack-aws-secret
+stringData:
+  creds: |
+    [default]
+    aws_access_key_id = test
+    aws_secret_access_key = test
+EOF
+{{</command>}}
+
+Finally, we create an AWS  `ProviderConfig` that references the secret created above, and defines a static `endpoint` pointing to the LocalStack URL `http://host.docker.internal:4566`:
+{{<command>}}
+$ cat <<EOF | kubectl apply -f -
+apiVersion: aws.upbound.io/v1beta1
+kind: ProviderConfig
+metadata:
+  name: default
+spec:
+  credentials:
+    source: Secret
+    secretRef:
+      name: localstack-aws-secret
+      namespace: default
+      key: creds
+  endpoint:
+    hostnameImmutable: true
+    # TODO: add more services to this list, as needed
+    services: [iam, s3, sqs, sts]
+    url:
+      type: Static
+      static: http://host.docker.internal:4566
+  skip_credentials_validation: true
+  skip_metadata_api_check: true
+  skip_requesting_account_id: true
+  s3_use_path_style: true
+EOF
+{{</command>}}
+
+{{<alert title="Note">}}
+The endpoint `http://host.docker.internal:4566` in the listing above assumes that you are running Kubernetes in the local Docker engine, and that LocalStack is up and running and available on default port `4566`.
+{{</alert>}}
+
+{{<alert title="Note">}}
+The Crossplane AWS provider currently requires us to specify the list of `services` for which the local `endpoint` is used as the target URL. Please make sure to extend this list accordingly if you're working with additional LocalStack services.
```

Comment:
consider adding
> The YAML above must configure the list of `services` since the Crossplane AWS provider requires it. 
Or something like that. 
I just read the `alert` and I was like "uh where is this referring to".